src/index.ts

```diff
@@ -15,7 +15,7 @@ import { join } from 'pathe'
 
 export { Options }
 
-export default createUnplugin<Options>((opt, meta) => {
+export default createUnplugin<Options>((opt = {}, meta) => {
```

Comment:
shouldn't we do 

```suggestion
export default createUnplugin<Options | undefined>((opt = {}, meta) => {
```

and avoid changing al the src/*.ts entry files like esbuild, rollup, etc

Reply:
yeah, it seems like this would create the correct type: https://github.com/unjs/unplugin/blob/main/src/types.ts#L69-L71